app/styles/libs/_helpers.scss

```diff
@@ -131,10 +127,24 @@ $spacer-heights: 50 100 200 300 400 500 600 700 800 900;
   width: fit-content;
 }
 
+.social-link-button {
+  width: 50px;
+  height: 50px;
+  max-width: 50px;
+  max-height: 50px;
+}
+
 .text-white {
   color: #fff;
 }
 
-.w-full {
```

Comment:
Why

Reply:
Oh sorry sir, Actually I forgot to add these changes in the commit, I removed them for testing purpose but forgot to add them again after it,  I will just add and then do the Pr, Thank you so much for reminding me sir.

Reply:
Sir I have added them again and the changes are still there , As you can see in the screenshot below:

![Screenshot (892)](https://user-images.githubusercontent.com/63748249/106020170-7adecb80-60e9-11eb-8a72-97d5c7ffcef9.png)